scripts/helmcharts/vars.yaml

```diff
@@ -20,6 +20,13 @@ postgresql: &postgres
 clickhouse:
   # For enterpriseEdition
   enabled: false
+  global:
+    clickhouse:
```

Comment:
I don't think we need this keys. You can directly add the keys under `clickhouse:` section
```
     clickhouseHost: clickhouse-openreplay-clickhouse.db.svc.cluster.local
     clickhousePort: 9000
```